services/core-api/app/api/mines/reports/models/mine_report_definition.py

```diff
@@ -60,9 +60,9 @@ def find_by_mine_report_definition_guid(cls, _id):
     @classmethod
     # TODO: We need to revisit how we return "static content" items via a "get active only" function. This means
     # that some content in the UI that depends on now "inactive" content will fail to render and can cause crashes.
-    def get_active(cls):
+    # the above comment may be not relevant anymore :)
```

Comment:
Can you please remove the TODO and your comment?

Reply:
sure